### ISSUE TITLE
wrong calculation of inverse matrix

### ISSUE THREAD

**Issue by pcerman** [grzegorzmazur/yacas#245]
release version 1.6.1
Inverse matrix of lower triangular matrix is calculated wrong. The same for upper triangular matrix.

**Comment by grzegorzmazur** [comment]
Good catch! The code for inverting triangular matrices apparently assumes unitriangular matrices and gives wrong result in other cases
